File: official/docs/golang/current/insurance/refund.go
package example

import (
	"fmt"

	"github.com/EasyPost/easypost-go/v5"
)

func refund() {
	client := easypost.New("EASYPOST_API_KEY")

	insurance, _ := client.RefundInsurance("ins_...")

	fmt.Println(insurance)
}

File: official/docs/java/v7/insurance/refund.java
package insurances;

import com.easypost.exception.EasyPostException;
import com.easypost.model.Insurance;
import com.easypost.service.EasyPostClient;

public class Retrieve {
    public static void main(String[] args) throws EasyPostException {
        EasyPostClient client = new EasyPostClient("EASYPOST_API_KEY");

        Insurance insurance = client.insurance.refund("ins_...");

        System.out.println(insurance);
    }
}
